edc-dataplane/edc-dataplane-azure-vault/build.gradle.kts

```diff
@@ -33,6 +33,10 @@ dependencies {
     }
     implementation(libs.edc.azure.identity)
     implementation("com.azure:azure-security-keyvault-secrets:4.6.2")
+    runtimeOnly(project(":edc-extensions:edr-cache-sql"))
```

Comment:
these should be declared in `:edc-dataplane-base`

Reply:
That would pull the sql implementation of the cache also in memory right?

Reply:
no, there isn't a dedicated `in-mem` dataplane, because the in-mem runtime has an embedded dataplane

Reply:
i meant also for testing then we would need to exclude the sql implementation for memory testing right?

Reply:
also the `runtime-memory` depends on `runtimeOnly(project(":edc-dataplane:edc-dataplane-base"))`

Reply:
Ah you're right. I would lean slightly toward leaving it in `edc-dataplane-base` (and excluding it for in-mem), since that is the "official" recommended module, and the in-mem one is only for testing.